Makefile

```diff
@@ -25,9 +25,14 @@ GRPC_DIR:=./vendor/google.golang.org/grpc
 install: submodule
 	go install $(PACKAGES)
 
+# Generate certs for unit and release tests.
+certs:
+	go run certgen/certgen.go
```

Comment:
replace by openssl command *and* make it the actual file/dependency so it doesn't keep doing it if the files do exist